apps/condo/domains/analytics/constants/restrictions.ts

```diff
@@ -0,0 +1,21 @@
+const MAX_DESCRIPTION_DISPLAY_LENGTH = 150
+const MIN_DESCRIPTION_LENGTH = 6
+const TICKET_CHART_PAGE_SIZE = 6
+const MAX_CHART_NAME_LENGTH = 35
+const MAX_CHART_LEGEND_ELEMENTS = 2
```

Comment:
it's better to move it to apps/condo/domains/analytics/utils/helpers.ts
it's not a domain level consts

Reply:
Done